### ISSUE TITLE
scale_robust_feature: run_scale_robust_feature_evaluation seed validation accepts bool/float/out-of-range seeds

### ISSUE THREAD

**Issue by rama0x1** [SlopDotCash/asi#779]
## Summary

\`run_scale_robust_feature_evaluation\` in \`alberta_framework/evaluation/scale_robust_feature.py\` (the executable protocol behind the registered \`scale_robust_pair_features\` evidence claim) validates its \`seeds\` argument with:

\`\`\`python
seed_schedule = tuple(int(seed) for seed in seeds)
if not seed_schedule:
    raise ValueError("seeds must be non-empty")
if len(set(seed_schedule)) != len(seed_schedule):
    raise ValueError("seeds must be unique")
if any(seed < 0 for seed in seed_schedule):
    raise ValueError("seeds must be non-negative")
\`\`\`

This is weaker than the codebase's own canonical seed-domain contract in \`alberta_framework/_seed_validation.py\` (\`require_unique_jax_seeds\`), which every other JAX-key-consuming benchmark entrypoint uses (e.g. \`upgd_ipmnist.run_ipmnist\`):

- \`int(seed)\` silently truncates a float (\`int(5.9) == 5\`) instead of rejecting it.
- \`int(seed)\` silently coerces \`bool\` (\`int(True) == 1\`) and numeric strings instead of rejecting them.
- There is no upper bound. Seeds are fed directly into \`jax.random.key(seed)\` via \`keys = jnp.stack([jr.key(seed) for seed in seeds])\`, and \`jax.random.key\` reduces an out-of-range seed modulo \`2**32\`. Confirmed live:

\`\`\`python
>>> import jax.random as jr
>>> jr.key_data(jr.key(5)).tolist() == jr.key_data(jr.key(5 + 2**32)).tolist()
True
\`\`\`

So two seeds recorded as distinct identities in the evidence artifact (e.g. \`5\` and \`4294967301\`) would silently execute the *identical* random stream — exactly the failure mode \`_seed_validation.py\`'s own docstring exists to prevent: "distinct caller values can execute as the same random stream while being recorded as different identities."

## Why this matters here

\`scale_robust_feature.py\` is a registered source for the \`scale_robust_pair_features\` accepted L2 evidence claim (see \`evaluation/evidence_manifest.py\`). \`run_scale_robust_feature_evaluation\` is public API (\`__all__\`) explicitly documented as usable for "development diagnostics" with alternate seed schedules, so this is a reachable, not merely theoretical, validation gap.

## Fix

Replace the inline validation with the existing shared helper \`alberta_framework._seed_validation.require_unique_jax_seeds\`, matching the pattern already used elsewhere in this codebase (e.g. \`upgd_ipmnist.py\`). Narrow, single-file change plus a new failing-test-first regression test (\`tests/test_scale_robust_feature.py\`, which did not exist before — only artifact/CLI tests existed and neither exercised this function directly).

No pinned \`outputs/\` artifact is touched. Editing this registered source file is expected to (and does) make \`scale_robust_pair_features\` report \`invalid\` in \`alberta-evidence-status\` until the frozen v2 protocol is rerun — this is the documented fail-closed design, not a regression: the registry was already fully \`invalid\` (exit 2) on a pristine \`origin/main\` checkout before this change, for unrelated pre-existing reasons.
